common/service/dynamicconfig/constants.go

```diff
@@ -94,6 +94,7 @@ var keys = map[Key]string{
 	FrontendThrottledLogRPS:               "frontend.throttledLogRPS",
 	EnableClientVersionCheck:              "frontend.enableClientVersionCheck",
 	ValidSearchAttributes:                 "frontend.validSearchAttributes",
+	SendRawWorkflowHistoryByDomain:        "frontend.sendRawWorkflowHistoryByDomain",
```

Comment:
please remove the ByDomain suffix just to be consistent with rest of the config i.e. we don't follow the pattern of suffixing with filter criteria